Cargo.toml

```diff
@@ -140,3 +140,6 @@ itertools = { version = "0.12", default-features = false }
 insta = "1.8"
 tempfile = "3.4"
 tikv-jemallocator = "0.5"
+
+[patch.crates-io]
+fuel-vm-private = { git = 'https://github.com/FuelLabs/fuel-vm.git', package = "fuel-vm", branch = "master" }
```

Comment:
I don't think we should commit patches to master.

Reply:
Patch removed [here](https://github.com/FuelLabs/fuel-core/pull/2211/commits/d944419bd3511adeaaa5ba15d0ae9e4d9fa70f39).


I added a note to the description saying that this PR is currently blocked and awaiting new `fuel-vm` release.